styles.css

```diff
@@ -1,64 +1,123 @@
 .body {
+  position: relative;
   margin: 0;
 }
+
 .wrapper {
   margin: auto;
   width: 80%;
 }
+
 .header{
   display: flex;
   justify-content: space-between;
+  margin-bottom: 100px;
 }
+
 .logo {
   position: relative;
-  margin-right: 63%;
   padding: 40px 0;
   padding-left: 44px;
   font: bold 24px/31px pt_serifbold, arial, helvetica;
 }
+
 .logo__circle {
- left: 0;
- top: 0;
- top: 50%;
- transform: translateY(-50%);
- width: 28px;
- height: 28px;
- position: absolute;
+  left: 0;
+  top: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  position: absolute;
 }
+
 .logo__circle_color_dark-blue {
- background: #0056EC;
- border-radius: 50%;
+  background: #0056ec;
+  border-radius: 50%;
 }
+
 .logo__circle_color_blue {
- left: 8px;
- top: 56%;
- background: #C8DBFB;
- border-radius: 50%;
- opacity: .7;
+  left: 8px;
+  top: 56%;
+  background: #c8dbfb;
+  border-radius: 50%;
+  opacity: .7;
 }
+
 .menu {
   display: flex;
   align-items: center;
 }
+
 .menu__item {
-  padding: 0 13px;
+  position: relative;
+  margin-right: 26px;
+  font: 18px/21px roboto, helvetica, arial, sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s width .5s;
+}
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+}
+
+.menu__item:hover {
+  opacity: .5;
 }
+
+.menu__item:hover::after {
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+}
+
+.menu__item:hover::after,
+.menu__item_active::after {
+  opacity: 1;
+  width: 100%;
+}
+
 .menu__item:last-child {
-  padding-right: 0;
+  margin-right: 0;
 }
-.a {
- font-family: Roboto;
+
+.main {
+  padding: 0 63% 100px 0;
+  background: center right no-repeat url(img/mainlmg..svg);
 }
-.main-content__title {
- font-family: PT serif;
- margin-bottom: 40px;
+
+.main__title {
+  position: relative;
+  font: 50px/65px pt_serifbold, arial, helvetica, sans-serif;
 }
-.main-content__subtitle{
-  margin-bottom: 40px;
+
+.main__subtitle {
+  margin: 40px 0 0 0;
+  font: 18px/21px roboto, arial, helvetica, sans-serif;
 }
-.p {
- font-family: Roboto;
+
+.main__title-text_highlight {
+  background: #ecf3ff;
 }
-.button {
- font-family: Roboto;
+
+.main__button {
+  display: inline-block;
+  margin-top: 41px;
+  font: 14px/16px Roboto, arial, helvetica, sans-serif;
+  color: #fff;
+  background: #0056ec;
+  padding: 14px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity  .5s;
 }
+
+.main__button:hover { opacity:  .5;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6259723)